webapp/src/components/modals/create_issue/create_issue_form.tsx

```diff
@@ -132,21 +133,32 @@ export default class CreateIssueForm extends React.PureComponent<Props, State> {
             this.setState(state);
         });
 
-        const fields = {
+        let fields = {
             summary: this.state.fields.summary,
             description: this.state.fields.description,
             project: {key: projectKey},
         } as CreateIssueFields;
 
-        const issueTypes = getIssueTypes(this.state.jiraIssueMetadata, projectKey);
-        const issueType = issueTypes.length ? issueTypes[0].id : '';
-        fields.issuetype = {
-            id: issueType,
-        };
+        if (fieldValues.issue_type) {
+            fields.issuetype = {
+                id: fieldValues.issue_type,
+            };
+        } else {
+            const issueTypes = getIssueTypes(this.state.jiraIssueMetadata, projectKey);
+            const issueType = issueTypes.length ? issueTypes[0].id : '';
```

Comment:
Please check whether we need 0 index every time

Reply:
yes